vermouth/pdb/pdb.py

```diff
@@ -130,7 +131,20 @@ def _skip(line, lineno=0):
     dbref1 = _skip
     dbref2 = _skip
     seqadv = _skip
-    seqres = _skip
+
+    def seqres(self, line, lineno=0):
+        """
+        Parse a SEQRES record. Store for processing later.
+
+        Parameters
+        ----------
+        line: str
+            The line to parse.
+
+        """
+        self._seqres.append(line)
```

Comment:
IMHO it would be better to deal with the syntax of the SEQRES line here, and indeed deal with the semantics in `do_seqres`